geometry_msgs/PoseStamped waypoint
bool blocking
---